.github/workflows/wheels-dependencies.sh

```diff
@@ -38,35 +60,44 @@ BZIP2_VERSION=1.0.8
 LIBXCB_VERSION=1.17.0
 BROTLI_VERSION=1.1.0
 
+function build_pkg_config {
+    if [ -e pkg-config-stamp ]; then return; fi
+    # This essentially duplicates the Homebrew recipe:
+    # https://github.com/Homebrew/homebrew-core/blob/master/Formula/p/pkg-config.rb
```

Comment:
```suggestion
    # This essentially duplicates the Homebrew recipe
```
https://github.com/Homebrew/homebrew-core/blob/master/Formula/p/pkg-config.rb no longer exists after https://github.com/Homebrew/homebrew-core/pull/194885

Reply:
Link rot before the PR even lands has to be a new record... :-)

I'll remove the reference.